DelvEdit/src/com/interrupt/dungeoneer/editor/ui/EditorUi.java

```diff
@@ -611,4 +611,10 @@ public void touchUp(int x, int y, int pointer, int button) {
             }
         }
     }
+
+    public void touchDown(int x, int y, int pointer, int button) {
```

Comment:
Only the `button` parameter is used, but the method signature is aligned with the counterpart `touchUp`.

Reply:
It is matching the interface of `InputProcessor` and I think this is perfectly fine to do.